github-release/README.md

```diff
@@ -25,6 +25,7 @@ github-release:
         generate-release-notes: true
         draft: true
         prerelease: true
+        binary-contents: path/to/binary
```

Comment:
we probably want this to be a list paths to assets to upload?

Reply:
Oh my *god* the `jq` in the `ubuntu-latest` image is *ancient*. Why do they make everything so hard 😭